doc/md/guides/mysql/cd-101.md

```diff
@@ -0,0 +1,156 @@
+---
+id: mysql-cd-101
+title: Detecting drops of foreign-key constraints on MySQL
+slug: /guides/mysql/checks/CD101
+tags: [linting, ci, destructive-change, mysql, drop-foreign-key]
+---
+
+## Introduction
+
+This document describes how Atlas's linter can be used to warn about dropping
+foreign key constraints from a MySQL database.
+
+Let's demonstrate by creating a database with 2 tables - users and pets:
+```sql
+CREATE TABLE users (
+	    id bigint NOT NULL AUTO_INCREMENT,
+	    name varchar(255) NOT NULL,
+	    PRIMARY KEY (id)
+	);
+
+CREATE TABLE pets (
+        id bigint NOT NULL AUTO_INCREMENT,
+        name varchar(255) NOT NULL,
+        owner_id bigint,
+        PRIMARY KEY (id),
+        CONSTRAINT owner_id FOREIGN KEY (owner_id) REFERENCES users (id)
+    );
+```
```

Comment:
Indentation is off. Also, did you generate that SQL or write it by hand?

Reply:
by hand

Reply:
ensure to uppercase the keywords to stay consistent with what Atlas generates and what all the other snippets look like :-)